gallery-so/gallery
# Trending -> Curated Feed

### Summary of Changes

- Replaces usage of the `trendingFeed` query with the new `curatedFeed` query.
Backend optimizations have been merged so this feed should be ready to go.

- also renames the navigation tab on the home page from "Trending" to "Curated" 
- also changes the url from "/trending" to "/home" 
- added a redirect from /trending to /home so the existing url doesn't break


### Demo or Before and After
Navbar before
![Screenshot 2023-08-08 at 16 34 11](https://github.com/gallery-so/gallery/assets/80802871/eb2f5cca-7836-43b6-9ae6-f19bdbb6eeef)

Navbar after
![Screenshot 2023-08-08 at 16 17 22](https://github.com/gallery-so/gallery/assets/80802871/85f2a9d2-04cb-42d0-a43d-db33d5c4e29a)


### Edge Cases

If signed out, the feed should still load properly with a non-personalized feed. 

### Testing Steps

Go to gallery's home page and check out the feed. It should load properly and show various feed events and posts. The Latest Feed and User Activity feeds should continue to load as expected.

### Checklist

Please make sure to review and check all of the following:
